.readthedocs.yml

```diff
@@ -9,16 +9,14 @@ build:
   os: "ubuntu-20.04"
   tools:
     python: "mambaforge-4.10"
-
-python:
-   install:
-      - requirements: docs/requirements.txt
-      - method: pip
-        path: .
+  jobs:
+    post_create_environment:
+      - pip install geocat-f2py
+      - python -m pip install --no-cache-dir .
```

Comment:
This shouldn't be necessary for viz. This step is in comp to make sure that the f2py dependency is installed prior to the environment solving for the geocat-comp install.

Reply:
I was wondering about this as well.

Reply:
Thanks. Removed it!